src/styles/blocks/catalog.scss

```
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  justify-content: center;
+  gap: $gap-size $gap-size;
```

if vertical end horizontal gaps are same you can write just one value
```suggestion
  gap: $gap-size
```